components/server/src/server.rs

```diff
@@ -662,10 +664,20 @@ impl<ER: RaftEngine> TiKVServer<ER> {
         let bps = i64::try_from(self.config.server.snap_max_write_bytes_per_sec.0)
             .unwrap_or_else(|_| fatal!("snap_max_write_bytes_per_sec > i64::max_value"));
 
+        let max_snapshot_file_raw_size = if self
```

Comment:
This will requires restart twice to make every node generate snapshot in multiple files.

Reply:
good point. fixed.

Reply:
> This will requires restart twice to make every node generate snapshot in multiple files.

good point. fixed.